### ISSUE TITLE
Any update for the coming of Big sur?

### ISSUE THREAD

**Issue by llity** [SaxMachine/Lenovo-Y50-70-OpenCore#4]
The big sur on the way. Thanks for your share an update for it.

**Comment by Destro706** [comment]
Big thx for the EFI. Tweaked it a bit for my needs/Maschine. 
Also waiting eagerly for the big sur efi. 
Can‘t get Uhd Screen to work.

**Comment by SaxMachine** [comment]
> Big thx for the EFI. Tweaked it a bit for my needs/Maschine. 
> Also waiting eagerly for the big sur efi. 
> Can‘t get Uhd Screen to work. 

Try to put framebuffers data that are on readme (recently updated).

**Comment by llity** [comment]
Thanks for your work. This PC has been working with me for 6 years.

**Comment by SaxMachine** [comment]
> Thanks for your work. This PC has been working with me for 6 years.
> 

You're welcome! 
Yeah me too, is a solid pc and is working better than many others. 
In this days I'm go to release the 0.6.3 for Catalina and Big Sur.
There are some good news for trackpad and the original Intel WiFi.

**Comment by Destro706** [comment]
> > Big thx for the EFI. Tweaked it a bit for my needs/Maschine.
> > Also waiting eagerly for the big sur efi.
> > Can‘t get Uhd Screen to work.
> 
> Try to put framebuffers data that are on readme (recently updated).

Tried the values in my catalina version. Sadly after adding them, screen stops working and I get only backlighted grey screen. 
Are they only suspected to work in Big Sur?

**Comment by SaxMachine** [comment]
> > > Big thx for the EFI. Tweaked it a bit for my needs/Maschine.
> > > Also waiting eagerly for the big sur efi.
> > > Can‘t get Uhd Screen to work.
> > 
> > Try to put framebuffers data that are on readme (recently updated).
> 
> Tried the values in my catalina version. Sadly after adding them, screen stops working and I get only backlighted grey screen. 
> Are they only suspected to work in Big Sur?

Have you set the DVMT-prealloc BIOS or via efi shell  to 128MB?

**Comment by Destro706** [comment]
> > > > Big thx for the EFI. Tweaked it a bit for my needs/Maschine.
> > > > Also waiting eagerly for the big sur efi.
> > > > Can‘t get Uhd Screen to work.
> > > 
> > > 
> > > Try to put framebuffers data that are on readme (recently updated).
> > 
> > 
> > Tried the values in my catalina version. Sadly after adding them, screen stops working and I get only backlighted grey screen.
> > Are they only suspected to work in Big Sur?
> 
> Have you set the DVMT-prealloc BIOS or via efi shell to 128MB?

Yes. Downgraded BIOS and set the prealloc via EFI Shell. It works with your Framebuffer Settings for 0.6.1 just fine on Catalina. It also did on Clover since Sierra.

Despite that I needed to add to your config.plist: 

<key>framebuffer-flags</key>
<data>AgAAAA==</data>
<key>framebuffer-fbmem</key>
<data>AAAAAw==</data>

to fix some random glitches on the uhd screen.

When I add your Values from readme the Screen stops working.
I updated the OC to 0.6.3 (I also migrated to VoodooPS2Controller to fix BrightnessKeys and made touchpad use of all Gestures), then updated to Big Sur with working Catalina config. After update everything works but internal screen. I can get external screens through hdmi to work and the graphics adapter shows the right memory (1536mb), but internal screen is not working. So I rolled back to Catalina.

**Comment by llity** [comment]
> > Thanks for your work. This PC has been working with me for 6 years.
> 
> You're welcome!
> Yeah me too, is a solid pc and is working better than many others.
> In this days I'm go to release the 0.6.3 for Catalina and Big Sur.
> There are some good news for trackpad and the original Intel WiFi.

I try to update to big sure with [EFI](https://github.com/GeekyCoder7/OpenCore-EFI-Lenovo-Y50-70), but the post-efi boot failed. the error code is OC:failed to load configuration.

**Comment by Destro706** [comment]
Finally Big Sur is working for me. 

I faced the same issue „OC: failed to load configuration“. 
After replacing the OC files with the debugging Version and then „updated“ to release Version of OC 0.6.4 the error disappeared. 

Also, for UHD Version i needed to compile and update to latest whatevergreen.kext. 
As a last step I needed to delete the EDID from config.plist and add a bootflag.

Now my Laptop runs latest 11.1 Big Sur without any flaws. I guess it even performs slightly better than Catalina. 

Will upload my final EFI today or tomorrow (when I have a little freetime). 

Thx once more to all people helping.
